contrib/tmux-sessionizer/template.py

```diff
@@ -0,0 +1,22 @@
+pkgname = "tmux-sessionizer"
+pkgver = "0.4.2"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = []
+makedepends = ["rust-std", "cargo", "pkgconf", "openssl-devel", "libgit2-devel"]
+pkgdesc = (
+    "Tmux Sessionizer: A tool for opening git repositories as tmux sessions"
+)
+maintainer = "Nova <froggo8311@proton.me>"
+license = "MIT"
+url = f"https://github.com/jrmoulton/{pkgname}"
+source = f"{url}/releases/download/v{pkgver}/source.tar.gz"
+sha256 = "3dc0a0206f3c67a2f8ffd65068916a8a802d5d79141189e272327205714aa82f"
+
+
+def do_install(self):
+    self.install_bin(f"target/{self.profile().triplet}/release/tms")
+
+
+def post_install(self):
+    self.install_license("LICENSE")
```

Comment:
```suggestion
def do_install(self):
    self.install_bin(f"target/{self.profile().triplet}/release/tms")
    self.install_license("LICENSE")
```

Reply:
can't we just drop the `install_bin` and `do_install` -> `post_install`?

Reply:
in this case, sure